ihmc-communication/src/main/java/us/ihmc/communication/controllerAPI/ControllerAPI.java

```diff
@@ -142,15 +142,11 @@ public static ROS2Topic<?> getBaseTopic(String controlModuleName, String robotNa
    public static <T> ROS2Topic<T> getTopic(ROS2Topic<?> baseTopic, Class<T> messageClass)
    {
       if (inputMessageClasses.contains(messageClass))
-      {
          return baseTopic.withInput().withTypeName(messageClass).withQoS(getQoS(messageClass));
-      }
-      if (outputMessageClasses.contains(messageClass))
-      {
+      else if (outputMessageClasses.contains(messageClass))
          return baseTopic.withOutput().withTypeName(messageClass).withQoS(getQoS(messageClass));
-      }
-
-      throw new RuntimeException("Topic does not exist: " + messageClass);
+      else
+         return baseTopic.withTypeName(messageClass).withQoS(getQoS(messageClass));
```

Comment:
without this else, it is not usable for messages that aren't included in the input/output message class lists.